recipes/dynare_preprocessor/meta.yaml

```diff
@@ -0,0 +1,40 @@
+package:
+  name: dynare_preprocessor
+  version: 0.0.1
```

Comment:
For packages without official releases, please ask the upstream project to make an official release. Otherwise, the version here may become out of sync with the upstream (canonical) version number.

If the upstream package is abandoned (it doesn't seem to be), then we have a convention that the package version should be something like "0.0.0.dev{{ build }}{{ git hash }}". I'd need to look up what we've done in the past, but the special build number + a run export prevents ABI breaks for non-versioned packages while maintaining version ordering.

Reply:
Yes, here I choose a very small number, to avoid any clash, but I'm happy to oblige, if there is 

There are two factors to consider here:
- last released version of upstream was 5.5 but hasn't switched to the new build system (meson). Based on this it would make sense to use number 5.6-dev .
- there is no separate release of the preprocessor which is a part of the whole distribution, so I don't know if it would follow a separate numbering.

I have no personal preference here, so I'll ask the package authors (I've been in touch with them before).

Reply:
Dynare upstream here.

We actually don’t publish releases of the Dynare preprocessor because it is only a component of Dynare, and so far we have never distributed it independently.

I think that using the version number of the whole Dynare could lead to problems, because you may want to update the preprocessor in Conda without waiting for a new Dynare release.

My suggestion would be to use a version number specific to Conda forge, that would incorporate the git hash. The scheme proposed by @carterbox looks good to me: `0.0.0.dev{{ build }}{{ git hash }}`. A slight variation could be `0.0.0.{{ date }}.{{ git hash }}`, which looks more explicit to me. But either is fine.

Reply:
I'm deeply indifferent between both versions. I'll go for the latest proposal by @svillemot unless @carterbox (or @SylvainCorlay) has a strong opinion.

Reply:
@carterbox : New version is `0.0.0.dev0.2a63840eb534c3dcd1f50477d935ecc92294f00b` . All good ?

Reply:
0.0.0.dev{{ build }}.{{ git hash }} should be fine.